features/gezag-minderjarige/1.3a-is-geboren-in-buitenland.feature

```diff
@@ -0,0 +1,78 @@
+#language: nl
+
+Functionaliteit: 1.3a - is geboren in het buitenland
+  Precondities voor deze vraag:
+  | 1.1 Staat persoon (minderjarige) als ingezetene in de BRP?             | Ja                                       |
+  | 1.2 Is persoon a. minderjarig en b. niet overleden?                    | Ja                                       |
+  | 1.3 Is minderjarige naar het buitenland geëmigreerd geweest?           | Nee                                      |
+
+  Een kind geboren in het buitenland kan het gezag uit het buitenland hebben meegenomen. Dit gezag kan niet betrouwbaar worden bepaald, omdat regelgeving uit het buitenland mogelijk niet bekend is bij de Nederlandse overheid.
+  
+  Een kind dat in het buitenland is geboren kan het gezag uit het buitenland hebben meegenomen. Het gezag kan niet betrouwbaar worden bepaald, omdat buitenlandse regelgeving van toepassing kan zijn die bij de Nederlandse overheid onbekend is. Uitzondering hierop zijn kinderen die in Nederland zijn geadopteerd.
+  
+  Hoe achterhalen:
+  Bepaal of de persoon als geboorteland een ander land heeft dan Nederland.
+  
+  Gebruikte velden:
+    - Geboorteland -> 01.03.30
+
+    Achtergrond:
+      Gegeven de persoon 'Regina' met burgerservicenummer '000000012'
+      * is meerderjarig
+      En de persoon 'Paul' met burgerservicenummer '000000024'
+      * is meerderjarig
+      En 'Regina' en 'Paul' zijn met elkaar gehuwd
+      En de persoon 'Nona' met burgerservicenummer '000000036'
+      * is ingeschreven in de BRP
+      * is minderjarig
+      * heeft 'Regina' als ouder 1
+      * heeft 'Paul' als ouder 2
+
+  Regel: Gezag wordt bepaald voor kinderen die in Nederland zijn geboren of geadopteerd
```

Comment:
geadopteerd hier weghalen? staat bij 1.3. b en hier staat geen scenario om dat te illusteren.

Reply:
Ik dacht daar eerst ook aan, maar heb het toch laten staan omdat de scenario

> Van een kind die in het buitenland is geboren en niet is geadopteerd in Nederland kan gezag niet worden bepaald

wel het tegendeel illustreert. Als je adoptie uit de rule haalt, dan komt adoptie in de scenario een beetje uit de lucht vallen. Als je adoptie uit zowel de rule als de scenario, dan komt de toelichting een beetje uit de lucht vallen. Thoughts?

Reply:
Ik sluit deze conversatie vanwege inactiviteit.